/builddir/bitfighter-022/bitfighter-022/discord-rpc/thirdparty/rapidjson-1.1.0/include/rapidjson/document.h:319:82: error: assignment of read-only member 'rapidjson::GenericStringRef<CharType>::length'
  319 |     GenericStringRef& operator=(const GenericStringRef& rhs) { s = rhs.s; length = rhs.length; }
--- a/bitfighter-022/discord-rpc/thirdparty/rapidjson-1.1.0/include/rapidjson/document.h
+++ b/bitfighter-022/discord-rpc/thirdparty/rapidjson-1.1.0/include/rapidjson/document.h
@@ -316,8 +316,6 @@
 
     GenericStringRef(const GenericStringRef& rhs) : s(rhs.s), length(rhs.length) {}
 
-    GenericStringRef& operator=(const GenericStringRef& rhs) { s = rhs.s; length = rhs.length; }
-
     //! implicit conversion to plain CharType pointer
     operator const Ch *() const { return s; }
 
